src/index.ts

```diff
@@ -245,12 +274,16 @@ const tracker: JupyterFrontEndPlugin<IDebugger> = {
       }
     });
 
-    palette.addItem({ command, category: 'Debugger' });
+    palette.addItem({ command: commandCreateDebugger, category: 'Debugger' });
+    palette.addItem({
+      command: commandStartDebugger,
+      category: 'Debugger'
+    });
```

Comment:
Since `palette` is passed as an optional dependency, we should first check the palette is present before adding items to it:

```typescript
if (palette) {
  palette.addItem(...);
}
```